File: data/imported/knowledge/slides/java/merged/snippets/snippet_015.java
@Test
    void testBentConnectorAdjustments() throws IOException {
        try (var pres = new Presentation()) {
            pres.getSlides().get(0).getShapes().clear();
            var conn = pres.getSlides().get(0).getShapes().addConnector(ShapeType.BENT_CONNECTOR3, 50, 50, 300, 200);
            if (conn.getAdjustments().size() > 0) {
                conn.getAdjustments().get(0).setRawValue(30000);
            }

            try (var pres2 = saveAndReopen(pres)) {
                // Find the connector shape
                Connector conn2 = null;
                var shapes2 = pres2.getSlides().get(0).getShapes();
                for (int i = 0; i < shapes2.size(); i++) {
                    IShape sh = shapes2.get(i);
                    if (sh instanceof Connector c) {
                        conn2 = c;
                        break;
                    }
                }
                assertThat(conn2).as("Connector not found after reload").isNotNull();
                if (conn2.getAdjustments().size() > 0) {
                    assertThat(conn2.getAdjustments().get(0).getRawValue()).isEqualTo(30000);
                }
            }
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_015.cs
[Fact]
    public void TestBentConnectorAdjustments()
    {
        // Adjustment values persist after save/reload.
        using var pres = new Presentation();
        var shapes = pres.Slides[0].Shapes!;
        shapes.Clear();
        var conn = shapes.AddConnector(ShapeType.BentConnector3, 50, 50, 300, 200);
        var adjustments = conn.Adjustments;
        if (adjustments is not null && adjustments.Count > 0)
        {
            adjustments[0].RawValue = 30000;
        }

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        // Find the connector shape
        Connector? conn2 = null;
        foreach (var sh in pres2.Slides[0].Shapes!)
        {
            if (sh is Connector c)
            {
                conn2 = c;
                break;
            }
        }
        conn2.Should().NotBeNull("Connector not found after reload");
        var adj2 = conn2!.Adjustments;
        if (adj2 is not null && adj2.Count > 0)
        {
            adj2[0].RawValue.Should().Be(30000);
        }
    }